app/services/decisions/client_decision_tree.rb

```diff
@@ -133,7 +131,7 @@ def determine_dwp_result_page
       DWP::UpdateBenefitCheckResultService.call(applicant)
 
       if applicant.passporting_benefit.nil?
-        edit(:retry_benefit_check)
+        edit('steps/dwp/cannot_check_dwp_status')
```

Comment:
@hiboabd What do you think of using  **:cannot_check_dwp_status** instead of string path

Reply:
`cannot_check_dwp_status` is in the dwp scope so i can't do that here 😩